<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>623</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>BrickGame</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/free-icon-tetris-720684.png</normaloff>:/free-icon-tetris-720684.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background: qlineargradient(x1:-0.2, y1:-0.2, x2:1.05, y2:1.05,
stop:0 white, stop: 0.8 rgb(52, 161, 235), stop:1 black) </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_right">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>505</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 30px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(227, 171, 0), stop:1 black) }
QPushButton:pressed {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 30px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(191, 144, 0), stop:1 black) 
}</string>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_action">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>510</y>
      <width>84</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 40px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(227, 171, 0), stop:1 black) }
QPushButton:pressed {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 40px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(191, 144, 0), stop:1 black) 
}
</string>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_up">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 30px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(227, 171, 0), stop:1 black) }
QPushButton:pressed {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 30px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(191, 144, 0), stop:1 black) 
}</string>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>505</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 30px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(227, 171, 0), stop:1 black) }
QPushButton:pressed {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 30px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(191, 144, 0), stop:1 black) 
}</string>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_down">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 30px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(227, 171, 0), stop:1 black) }
QPushButton:pressed {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 30px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(191, 144, 0), stop:1 black) 
}</string>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>447</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 20px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(227, 171, 0), stop:1 black) }
QPushButton:pressed {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 20px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(191, 144, 0), stop:1 black) 
}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pause">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>447</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 20px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(227, 171, 0), stop:1 black) }
QPushButton:pressed {
border: 2px solid #094065;
border-width: 2px;
padding: 1px;
border-style: solid;
border-radius: 20px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 white, stop: 0.8 rgb(191, 144, 0), stop:1 black) 
}</string>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="gamename">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>148</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(0,0,0)</string>
    </property>
    <property name="text">
     <string>BrickGame</string>
    </property>
   </widget>
   <widget class="QFrame" name="gamefield">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>298</width>
      <height>410</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:  rgb(158, 173, 134)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>81</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="hi_score_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>Hi-Score</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="hi_score_number">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
border-width: 0px;
padding: 1px;
border-style: solid;
color: rgb(0,0,0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="score_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>Score</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="score_number">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
border-width: 0px;
padding: 1px;
border-style: solid;
color: rgb(0,0,0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="level_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>Level</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="level_number">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
border-width: 0px;
padding: 1px;
border-style: solid;
color: rgb(0,0,0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="speed_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>Speed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="speed_number">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
border-width: 0px;
padding: 1px;
border-style: solid;
color: rgb(0,0,0);</string>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="next_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>342</y>
       <width>80</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="pause_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>p - pause</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="exit_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(0,0,0)</string>
        </property>
        <property name="text">
         <string>q - exit</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
